### PR TITLE
[core] Add a CLI command for listing out deployed GraphQL endpoints

### PR DESCRIPTION
**Pull request by saasen**
**Type of change (check at least one)**

- [ ]  Bug fix (non-breaking change which fixes an issue)
- [x]  New feature (non-breaking change which adds functionality)
- [ ]  Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ]  Documentation (fix or update to documentation)
- [ ]  Maintenance
- [ ]  Other, please describe:

**Does this change require a documentation update? (Check one)**

- [x]  Yes
- [ ]  No

**Current behavior**

After we introduced multiple GraphQL endpoints, there are currently no way of knowing what you already deployed (unless you take notes).

**Description**

This PR adds functionality for listing out currently deployed GraphQL endpoints.

**Note for release**

Add a CLI GraphQL sub-command for listing out deployed GraphQL endpoints.

**Checklist** 

- [x]  I have read the [Contributing Guidelines](https://github.com/sanity-io/sanity/blob/next/CONTRIBUTING.md)
- [ ]  The PR title includes a link to the relevant issue
- [x]  The PR title is appropriately formatted: `[some-package] PR title (#123)`
- [x]  The code is linted
- [x]  The test suite is passing
- [ ]  Corresponding changes to the documentation have been made

### CODE REVIEW

File: packages/@sanity/core/src/actions/graphql/listApisAction.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    })
+  } catch (err) {
+    if (err.statusCode === 404) {
+      endpoints = []
+    } else {
+      throw err
+    }
+  }
+
+  endpoints = [{
```
Comment (vicbergquist) [inline packages/@sanity/core/src/actions/graphql/listApisAction.js]: Forgot to remove the mock endpoints again?

Comment (saasen) [reply packages/@sanity/core/src/actions/graphql/listApisAction.js]: 🤦‍♀
I guess that's what you get when you don't have tests on this. I'll fix this right away!

File: packages/@sanity/core/src/commands/graphql/listGraphQLAPIsCommand.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import lazyRequire from '@sanity/util/lib/lazyRequire'
+
+const helpText = `
+Options
+  N/A
+
```
Comment (rexxars) [inline packages/@sanity/core/src/commands/graphql/listGraphQLAPIsCommand.js]: This should be removed to align with other flag-less commands

File: packages/@sanity/core/src/actions/graphql/listApisAction.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  })
+
+  let endpoints
+  try {
+    endpoints = await client.request({
+      url: `/apis/graphql`,
+      method: 'GET'
+    })
+  } catch (err) {
+    if (err.statusCode === 404) {
```
Comment (rexxars) [inline packages/@sanity/core/src/actions/graphql/listApisAction.js]: I don't think this should be a 404 from the backend unless it means the route/project doesn't exist

Comment (saasen) [reply packages/@sanity/core/src/actions/graphql/listApisAction.js]: Yep. Let's discuss that in the backend PR.

Comment (saasen) [reply packages/@sanity/core/src/actions/graphql/listApisAction.js]: I'll just remove that check for now.

**Pull request merged**